### PR TITLE
Real recconection

### PR DESCRIPTION
**Pull request by idanstark42**

### CODE REVIEW

File: client/mhub-listener.js [hunk 1]
```diff
@@ -2,28 +2,47 @@
...
 const mClient = new MClient(`ws://${window.location.hostname}:13900`)
 
 const listeners = {}
 
 let connectPromise = null
 
-mClient.on('close', () => {
+function attemptRecconection () {
   connectPromise = null
+  console.log('Disonnected from mhub')
```
Comment (2roy999) [inline client/mhub-listener.js]: use logger

File: client/mhub-listener.js [hunk 2]
```diff
@@ -2,28 +2,47 @@
...
 function connect () {
   if (!connectPromise) {
     connectPromise = Promise.resolve(mClient.connect())
       .then(() => {
-        mClient.on('message', message => {
-          const topic = message.topic
-
-          listeners[topic].forEach(listener => listener(message.data))
-        })
+        console.log('Connected to mhub')
```
Comment (2roy999) [inline client/mhub-listener.js]: use logger

Comment (idanstark42) [reply client/mhub-listener.js]: This is client, how could I use the logger?

**Comment by 2roy999** [comment]
You are right. My wrong

**Comment by alanggreen** [comment]
@Idan Stark <idanstark42@gmail.com> Didn't you say that the client could
sent the log text to the server for logging?

**Pull request merged**